db/views/organization_capacities_v03.sql

```diff
@@ -0,0 +1,21 @@
+WITH
+    organization_id_by_vita_partner_id AS (
+        select id, (CASE WHEN parent_organization_id IS NULL THEN id ELSE parent_organization_id END) AS organization_id
+        from vita_partners
+    ),
+    client_ids AS
+    (
+        select distinct client_id
+        from tax_returns
+
+        where tax_returns.status >= 102 AND tax_returns.status <= 404 AND tax_returns.status != 403 AND tax_returns.status != 106 AND tax_returns.status != 130
```

Comment:
copy of `organization_capacities_v02` with `AND tax_returns.status != 130` added